python/tests/test_tl_sign.py

```diff
@@ -0,0 +1,82 @@
+import pytest
```

Comment:
Seems to be missing a few scenarios
* signature_no_headers
* signature_method_mismatch
* signature_path_mismatch
* signature_header_mismatch
* signature_body_mismatch
* signature_missing_signature_header
* signature_required_header_missing_from_signature
* flexible_header_case_order_verify
* extract_kid / extract_jku (or whole jws header however that's done here)
* verify_with_jwks